test/TorchSharpTest/FactIgnoreOnPlattformAttribute.cs

```diff
@@ -0,0 +1,43 @@
+﻿#nullable enable
+using System.Linq;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace TorchSharp
+{
+    /// <summary>
+    /// To ignore an xUnit <see cref="FactAttribute">Fact</see> on a given platform or architecture.
+    /// </summary>
+    public sealed class FactIgnoreOnPlattformAttribute : FactAttribute
```

Comment:
Oh, yes! This will be very useful. The other skip-test attribute I've thought about is one that tests whether CUDA is available.

Reply:
@MovGP0 -- I just wanted to let you know that after next week, I'll be taking a break until January 2nd.

Reply:
@NiklasGustafsson I am currently training for the national championships and got some additional training schedules on weekends; so there won't be progress in the next two weeks. Hope I can get some work done by the end of the month. Enjoy your break till then!

Reply:
@MovGP0 -- that sounds exciting! National championship in what?

Reply:
Standard Dance Formation